applications/CSharpWrapperApplication/tests/cpp_tests/test_calculate.cpp

```diff
@@ -22,13 +22,13 @@
 #include "testing/testing.h"
 #include "containers/model.h"
 #include "custom_includes/kratos_wrapper.h"
-#include "utilities/os_utilities.h"
+#include "includes/kratos_filesystem.h"
 
 namespace Kratos {
     namespace Testing {
         void CreateMDPAFile() {
             std::filebuf buffer;
-            buffer.open(OSUtilities::GetCurrentWorkingDir() + "/file.mdpa", std::ios::out);
+            buffer.open(FilesystemExtensions::JoinPaths({FilesystemExtensions::CurrentWorkingDirectory(), "/file.mdpa"}), std::ios::out);
```

Comment:
```suggestion
            buffer.open(FilesystemExtensions::JoinPaths({FilesystemExtensions::CurrentWorkingDirectory(), "file.mdpa"}), std::ios::out);
```

`FilesystemExtensions::JoinPaths` adds the separator

Reply:
Ups